From a6293f48a0deb53044ecf593a5ed0880c7f1f128 Mon Sep 17 00:00:00 2001
From: Matt Madison <matt@madison.systems>
Date: Thu, 20 May 2021 05:49:34 -0700
Subject: [PATCH] Internet connectivity irrelevant for pre-fetched firmware

With bitbake fetching the firmware in advance, we
do not need the internet connectivity check during
the build.

Signed-off-by: Matt Madison <matt@madison.systems>
---
 CMakeLists.txt | 13 -------------
 1 file changed, 13 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1ddedfb18..edbb5a9ab 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -3,7 +3,6 @@ cmake_minimum_required(VERSION 3.1.0)
 project(librealsense2 LANGUAGES CXX C)
 
 include(CMake/lrs_options.cmake)
-include(CMake/connectivity_check.cmake)
 
 #Deprecation message, should be removed in future releases
 if(${FORCE_LIBUVC} OR ${FORCE_WINUSB_UVC} OR ${ANDROID_USB_HOST_UVC})
@@ -11,18 +10,6 @@ if(${FORCE_LIBUVC} OR ${FORCE_WINUSB_UVC} OR ${ANDROID_USB_HOST_UVC})
     set(FORCE_RSUSB_BACKEND ON)
 endif()
 
-# Checking Internet connection, as TM2 needs to download the FW from amazon cloud
-if(BUILD_WITH_TM2 AND NOT INTERNET_CONNECTION)
-    message(WARNING "No internet connection, disabling BUILD_WITH_TM2")
-    set(BUILD_WITH_TM2 OFF)
-endif()
-
-# Checking Internet connection, as DEPTH CAM needs to download the FW from amazon cloud
-if(IMPORT_DEPTH_CAM_FW AND NOT INTERNET_CONNECTION)
-    message(WARNING "No internet connection, disabling IMPORT_DEPTH_CAM_FW")
-    set(IMPORT_DEPTH_CAM_FW OFF)
-endif()
-
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR}/CMake)
 
 # include librealsense general configuration
-- 
2.30.2

